### PR TITLE
Update classnames pkg to latest version

### PR DESCRIPTION
**Pull request by bsteinbk**
Update classnames pkg to latest version. Since newest version include type definitions we no longer need types package.

### CODE REVIEW

**Comment by pmconne** [comment]
Can you hold off until I integrate the new addon? If you'll recall I had to lock the package version on imodel02.

**Comment by bsteinbk** [comment]
@pmconne I only updated the package due to the issues you saw.  I will not merge until your addon merges.

**Review by DanEastBentley — APPROVED** [review]

**Review by pmconne — APPROVED** [review]

**Review by mdastous-bentley — APPROVED** [review]

**Pull request merged**
